etc/Makefile.am

```diff
@@ -0,0 +1,3 @@
+#if HAVE_SYSTEMD
```

Comment:
Commit message typo: "interacted with systemd" :arrow_right: "controlled with systemd" or "started with systemd"?

Reply:
Thanks for catching this! Just force-pushed a fix for this commit message.